changelog.d/20240812_161734_mzhiltso_job_chunks.md

```diff
@@ -0,0 +1,4 @@
+### Changed
+
+- Jobs now have separate chunk ids starting from 0, instead of using ones from the task
```

Comment:
I would suggest to formulate somehow more clear for end user (not sure actually how exactly). 
But this record does not say me (as a CVAT user anything). This is probably only important for REST API users 

Maybe: **Numeration of data chunks in any job always starts with 0**

Reply:
By the way this change is breaking. Thus it means that we have to up major version of REST API. 
Hovewer I am not sure we want do it now. Probably worthy of internal discussion

Reply:
Why did you decide so?

Reply:
We never established SemVer for the server component. There were discussions a couple of years ago, we decided to postpone it. Currently, we can have any changes in the server API in any release, the version is the same as in the release itself.

Reply:
We increase server version each release (minor or patch) depends on changes. CVAT version = its server version

Reply:
So your suggestion is to make it CVAT 3.0?

Reply:
As I mentioned in the beginning: 

> Hovewer I am not sure we want do it now. Probably worthy of internal discussion

As I see strong reasons do not update version right now.

Reply:
Maybe instead of modifying existing `chunk_number` parameter, we have to add one more and deprecate previous behaviour

Reply:
Added the `index` parameter.